kernel/src/actions/deletion_vector.rs

```diff
@@ -101,10 +101,17 @@ impl DeletionVectorDescriptor {
     ) -> DeltaResult<RoaringTreemap> {
         match self.absolute_path(parent)? {
             None => {
-                let bytes = z85::decode(&self.path_or_inline_dv)
+                let byte_slice = z85::decode(&self.path_or_inline_dv)
                     .map_err(|_| Error::deletion_vector("Failed to decode DV"))?;
-                RoaringTreemap::deserialize_from(&bytes[12..])
-                    .map_err(|err| Error::DeletionVector(err.to_string()))
+                let magic = slice_to_u32(&byte_slice[0..4], Endian::Little)?;
```

Comment:
Shouldn't this be 0..3 ? First four bytes right?

Reply:
`0..4` is the first 4 bytes, `0..=3` would work too :)

Reply:
oh yes, inclusive/exclusive somewhat non intuitive